textdb/textdb-textql/src/main/java/edu/uci/ics/textdb/textql/statements/CreateViewStatement.java

```diff
@@ -101,6 +104,25 @@ public String getOutputNodeID(){
     public List<String> getInputViews(){
         return Arrays.asList(subStatement.getId());
     }
+
+    /**
+     * Build the resulting output schema of this predicate based on the given input schemas.
+     * This statement does not apply any modification to the input schema, thus the output
```

Comment:
Improve the comments to make it clear that the input of CreateView is a single schema.